lib/HARToPostmanCollectionBodyMapper.js

```diff
@@ -157,18 +157,18 @@ function isMimeType(mimeType, expectedMimeType) {
  * @returns {array} the list of parameters in the url
  */
 function getParamsFromEncoded(harRequest) {
-  const urlParamsArray = harRequest.postData.params,
-    urlParamsAsText = harRequest.postData.text;
+  const urlParamsArray = harRequest?.postData?.params,
+    urlParamsAsText = harRequest?.postData?.text;
   let params = [];
-  if (urlParamsArray.length > 0) {
+  if (urlParamsArray?.length > 0) {
     params = urlParamsArray.map((param) => {
       return {
         key: param.name,
         value: param.value
       };
     });
   }
-  else if (urlParamsAsText) {
+  else if (typeof urlParamsAsText === 'string') {
```

Comment:
is urlParamsAsText always expected to be of type string?

Reply:
@agrwl-harsh Yes, we're directly using `.split()` function next line from this which should be only done if it's string to avoid typeerrors.